app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -10,7 +10,8 @@ class ManageIQ::Providers::Openshift::ContainerManager < ManageIQ::Providers::Co
   has_many :container_routes,                      :foreign_key => :ems_id, :dependent => :destroy
   has_many :container_projects,                    :foreign_key => :ems_id, :dependent => :destroy
 
-  default_value_for :port, 8443
+  DEFAULT_PORT = 8443
+  default_value_for :port, DEFAULT_PORT
```

Comment:
> I wonder if this statement which appears twice (in k8s and ose classes) could be moved to the ContainerProviderMixin?

@abonas Its a little confusing without ```DEAFAULT_PORT``` next to it. 
How about having it appear just in k8s container manager since openshifts container manager inherits it?